tests/config.yaml

```diff
@@ -34,9 +34,11 @@ bootstrap:
     eligibility-confidence-param: '5'
     eligibility-epoch-offset: '0'
     genesis-active-size: '50'
+    executable-path: '/bin/go-spacemesh'
```

Comment:
why is this added?

Reply:
this goes to the harness.
it runs with various executables so every suite passes the matching one.